sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Table.cs

```diff
@@ -548,7 +583,7 @@ public static bool TryLoadTable(IAmazonDynamoDB ddbClient, string tableName, boo
         /// </returns>
         public static bool TryLoadTable(IAmazonDynamoDB ddbClient, string tableName, DynamoDBEntryConversion conversion, bool isEmptyStringValueEnabled, out Table table)
```

Comment:
Why are we not treating this function similar to the "LoadTable" function and giving the user a way to set the caching mode?

Reply:
+1

Reply:
This was just a miss, I tend to think of just `LoadTable` and forgot we had this one.

Addressed in https://github.com/aws/aws-sdk-net/compare/8a4f5a780d437debec1f38696b9dc2f069b991c1..1161dc30f02a5a18108c28b0191f21da5080a368